src/pages/apps/Read.tsx

```diff
@@ -76,51 +76,48 @@ export default function ReadApp() {
-              }}>
-              <Typography variant="h3" sx={{margin: '5px 40px 0px 10px'}}>
-                {app.name}
-              </Typography>
-              <Typography variant="h5" sx={{margin: '5px 40px 0px 10px'}}>
-                {app.description}
-              </Typography>
-              {app.active_app_tags ? (
-                <Box>
-                  {app.active_app_tags.map((tagMap) => (
-                    <Chip
-                      key={'tag' + tagMap.active_tag!.id}
-                      label={tagMap.active_tag!.name}
-                      color="primary"
-                      onClick={() => navigate(`/tags/${tagMap.active_tag!.name}`)}
-                      icon={<TagIcon />}
-                      sx={{
-                        margin: '2px',
-                        marginTop: '5px',
-                        bgcolor: (theme) => (tagMap.active_tag!.enabled ? 'primary' : theme.palette.action.disabled),
-                      }}
-                    />
-                  ))}
-                </Box>
-              ) : null}
-              <Stack style={{position: 'absolute', right: '10px'}}>
-                <Tooltip title="Edit" placement="right" PopperProps={moveTooltip}>
-                  <div>
-                    <CreateUpdateApp currentUser={currentUser} app={app} />
-                  </div>
-                </Tooltip>
-                <Tooltip title="Delete" placement="right" PopperProps={moveTooltip}>
-                  <div>
-                    <DeleteApp currentUser={currentUser} app={app} />
-                  </div>
-                </Tooltip>
+            <Paper sx={{py: 2, px: 2}}>
+              <Stack direction="column" gap={2}>
+                <Stack alignItems="center" direction="column" gap={1}>
+                  <Typography variant="h3" textAlign="center">
+                    {app.name}
+                  </Typography>
+                  <Typography variant="h5" textAlign="center">
+                    {app.description}
+                  </Typography>
+                  {app.active_app_tags ? (
+                    <Box>
+                      {app.active_app_tags.map((tagMap) => (
+                        <Chip
+                          key={'tag' + tagMap.active_tag!.id}
+                          label={tagMap.active_tag!.name}
+                          color="primary"
+                          onClick={() => navigate(`/tags/${tagMap.active_tag!.name}`)}
+                          icon={<TagIcon />}
+                          sx={{
+                            margin: '2px',
+                            marginTop: '5px',
+                            bgcolor: (theme) =>
+                              tagMap.active_tag!.enabled ? 'primary' : theme.palette.action.disabled,
+                          }}
+                        />
+                      ))}
+                    </Box>
+                  ) : null}
+                </Stack>
+                <Divider />
```

Comment:
When the current user isn't an Access Admin (member of App-Access-Owners) or isn't an App Owner (owner of App Owners group), they don't see anything below this divider. Is there some way we could remove it for that case?

![Screenshot 2024-11-01 at 1 42 25 PM](https://github.com/user-attachments/assets/f3e1b3e1-2459-45c4-9718-90e545fe7afd)

Reply:
Ahhh good point. I'll update there and in tags

Reply:
updated!